src/diffusers/__init__.py

```diff
@@ -60,6 +60,7 @@
 
 if is_torch_available() and is_transformers_available():
     from .pipelines import (
+        LDMSuperResolutionPipeline,
```

Comment:
We should also add a doc page for this under 

`docs/source/api/pipelines/ldm_superresolution.mdx

Reply:
If you have some time @duongna21, it'd be super nice if you could add it to the docs - otherwise happy to help out :-)

Reply:
Do we need `transformers` for the model? I think we can just add it under `if is_torch_available()`